[KodiProps] Fix multi wrapper separator

knaerzche opened the pull request:
According to the wiki, the separator for multiple wrappers is `,`, not `+`.

## Description
`ADP::KODI_PROPS::CCompKodiProps::ParseDrmOldProps` currently builds multiple wrappers with `+` as seperator which makes it unparsable for `HelperWv::TranslateWrapper` .

## Motivation and context
Fix a bug.

## How has this been tested?
With a stream using `json,base64` (`JB`)

## Screenshots (if appropriate):

## Types of change
<!--- What type of change does your code introduce? Put an `x` in all the boxes that apply like this: [X] -->
- [ x] **Bug fix** (non-breaking change which fixes an issue)
- [ ] **Clean up** (non-breaking change which removes non-working, unmaintained functionality)
- [ ] **Improvement** (non-breaking change which improves existing functionality)
- [ ] **New feature** (non-breaking change which adds functionality)
- [ ] **Breaking change** (fix or feature that will cause existing functionality to change)
- [ ] **Cosmetic change** (non-breaking change that doesn't touch code)
- [ ] **None of the above** (please explain below)

## Checklist:
<!--- Go over all the following points, and put an `X` in all the boxes that apply like this: [X] -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [x ] I have read the **[Contributing](https://github.com/xbmc/xbmc/blob/master/docs/CONTRIBUTING.md)** document
- [ x] My code follows the **[Code Guidelines](https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md)** of this project 
- [ ] My change requires a change to the Wiki documentation
- [ ] I have updated the documentation accordingly

CastagnaIT reviewed (changes requested):
this change is wrong,
as is also explained on wiki, the code its correct
i just forgot to fix the above method comment

knaerzche commented:
It's not like I scanned the comments in this code and adapted the code to the comment: Playing a stream with the mentioned "unwrapping method" fails with: `Cannot translate license wrapper, unknown type json+base64` and this change fixes it. I didn't look too deep in this code, but I'd guess it comes from here: https://github.com/xbmc/inputstream.adaptive/blob/8720557ed939ba5d4f5e227b052e3daad9e2f3e3/src/CompKodiProps.cpp#L520 - it uses `+` as seperator, not `,`

CastagnaIT commented:
yeah good catch
please fix all them with `,`, and also the above method comment

knaerzche committed:
[KodiProps] Fix multi wrapper separator

According to the wiki, the separator for multiple wrappers is ',', not '+'.
Fix it.

Signed-off-by: Alex Bee <knaerzche@gmail.com>

knaerzche commented:
Done. (Took some time for me to find the wiki entry you were referring to)

CastagnaIT reviewed (approved):
well done
thank you for your first contrib
